build.gradle

```diff
@@ -438,6 +465,11 @@ dependencies {
     datagenmainImplementation fg.deobf("curse.maven:i-like-wood-biomes-o-plenty-437203:${ilikewood_bop_id}")
     datagenmainImplementation fg.deobf("curse.maven:i-like-wood-oh-the-biomes-youll-go-475110:${ilikewood_byg_id}")
     datagenmainRuntimeOnly fg.deobf("curse.maven:projecte-226410:${projecte_id}")
+
+    datagenmainCompile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    datagengeneratorsCompile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    datagenadditionsCompile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    datagentoolsCompile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
```

Comment:
strictly speaking this should also probably be added to datagendefenseCompile even if it is just a dummy module currently